sdk/src/main/java/dev/cdevents/CDEvents.java

```diff
@@ -97,17 +90,17 @@ public static boolean validateCDEvent(CDEvent cdEvent) {
 
     /**
      * Creates cdEvent from cdEventJson string and validates against schema.
-     * @param cdEventJson
-     * @return CDEvent, needs type casting to specific CDEvent class
+     * @param cdEventJson json string of any CDEvent type
+     * @return CDEvent needs type casting to specific CDEvent class
      */
     public static CDEvent cdEventFromJson(String cdEventJson) {
-        if (!validateCDEventJson(cdEventJson)) {
-            throw new CDEventsException("CDEvent Json validation failed against schema");
-        }
         String eventType = getUnVersionedEventTypeFromJson(cdEventJson);
         CDEventConstants.CDEventTypes cdEventType = getCDEventTypeEnum(eventType);
         try {
-            CDEvent cdEvent = (CDEvent) new ObjectMapper().readValue(cdEventJson, cdEventType.getEventClass());
+            CDEvent cdEvent = new ObjectMapper().readValue(cdEventJson, cdEventType.getEventClass());
```

Comment:
Why are we constructing a new object mapper every time?
Is this method only ever called once? Even so we probably should move it to a field

Reply:
Removed this method using the existing APIs